pkg/hypervisor/hypervisor.go

```diff
@@ -128,11 +133,15 @@ func (m *Hypervisor) AddMockData(config MockConfig) error {
 // ServeHTTP implements http.Handler
 func (m *Hypervisor) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	r := chi.NewRouter()
-
-	r.Use(middleware.Timeout(httpTimeout))
 	r.Use(middleware.Logger)
 
 	r.Route("/api", func(r chi.Router) {
+		r.Use(middleware.Timeout(httpTimeout))
+
+		r.Get("/ping", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is there a reason why it is not located in the group with all APIs below?

Reply:
I feel it doesn't need the authentication cookie so I have kept it separate.

Reply:
@evanlinjin Fair enough. I would just extract the body to a method to reduce `ServeHTTP`'s length.